ACE/ace/Log_Msg.cpp

```diff
@@ -80,11 +82,11 @@ class ACE_Msg_Log_Cleanup: public ACE_Cleanup_Adapter<ACE_Log_Msg>
 
 #if defined (ACE_WIN32) && !defined (ACE_HAS_WINCE) && !defined (ACE_HAS_PHARLAP)
 #  define ACE_LOG_MSG_SYSLOG_BACKEND ACE_Log_Msg_NT_Event_Log
+#elif defined (ACE_ANDROID)
+#  define ACE_LOG_MSG_SYSLOG_BACKEND ACE_Log_Msg_Android_Logcat
 #elif !defined (ACE_LACKS_UNIX_SYSLOG) && !defined (ACE_HAS_WINCE)
 #  define ACE_LOG_MSG_SYSLOG_BACKEND ACE_Log_Msg_UNIX_Syslog
-#else
```

Comment:
Is the removal of the else part intended?

Reply:
It was because it's not used. `ACE_LOG_MSG_SYSLOG_BACKEND` was only used if the conditions above were true and `SYSLOG` was enabled, other wise it created `ACE_Log_Msg_IPC` but calls it by that name, not `ACE_LOG_MSG_SYSLOG_BACKEND`. I mainly did it because it simplifies the check below and it only has to check if `ACE_LOG_MSG_SYSLOG_BACKEND` is defined.